private/tools/java/rules/jvm/external/jar/BUILD

```diff
@@ -10,6 +10,23 @@ java_binary(
     ],
 )
 
+java_library(
+    name = "ListPackages-lib",
+    srcs = ["ListPackages.java"],
+    deps = [
+        "@maven//:com_google_code_gson_gson",
```

Comment:
You don't want to depend on the default `@maven` namespace here, since it means that all users of `rules_jvm_external` will need to add gson to their dependencies. Use `@rules_jvm_external_deps` which is created in `//:repositories.bzl`

Reply:
Fixed - thanks!